hi-ml-cpath/src/health_cpath/models/transforms.py

```diff
@@ -312,3 +312,56 @@ def __call__(self, data: Mapping) -> Mapping:
         with elapsed_timer(message):
             out_data = self.transform(data)
         return out_data
+
+
+class NormalizeBackgroundd(MapTransform):
+    """Normalize the background of an image by deviding the image by the background intensity. The background
+    intensities are either defined by the user or are computed as the q-th percentile of the image."""
```

Comment:
Maybe you could add what effect this transform has? You're saying how it is computed - but what happens to pixels that have the background value after the transform is applied?

Reply:
thank you. I have now updated the docs